assets/js/components/indicators.js

```diff
@@ -288,12 +288,18 @@ export class IndicatorSection {
     this.comparisonButtonsContainer.insertBefore(this.chartContainer);
   }
 
+  highlightComparisonButton(section, muniId) {
+    $(`${section} .chart-btn`).removeClass('active');
+    $(`${section} #${muniId}`).addClass('active');
+  }
+
   _updateComparisonButtons() {
     this.comparisonButtonsContainer.empty();
     this.comparisonButtonsContainer.append('Comparing ');
     [this.chartData(), ...(this.comparisons)].forEach((comparison) => {
-      const button = $(' <button class="button" style="display: unset; margin: 2px 2px"></button> ');
+      const button = $(` <button id="${comparison.municipality.code}"class="button chart-btn"></button> `);
```

Comment:
it's not valid to have more than one element in a document with the same id. So if you want to identify a button by the muni code, your options are

 - generate unique IDs, e.g. `#x-y-z-section-muni-BUF` - a bit messy
 - make a class out of the muni code, e.g. `.muni-code-BUF`
 - set the muni code as a data attribute and select on that, e.g. `<button data-muni="BUF">`

you can select on data attributes with `document.querySelector('[data-id="box1"]` or `$("ul[data-group='Companies'] li[data-company='Microsoft']")`

I think the data attribute one is the most appropriate for muni codes

Reply:
Changed